### ISSUE TITLE
can't access autoload gdscript in c#

### ISSUE THREAD

**Issue by theBSH** [godotengine/godot#85318]
### Godot version

4.1.2

### System information

Godot v4.1.2.stable.mono - Windows 11 22h2 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3060 (NVIDIA; 31.0.15.3742) - 11th Gen Intel(R) Core(TM) i5-11600K @ 3.90GHz (12 Threads)

### Issue description

i cant access the global variables I want to set in my c# code nor can I access the functions

### Steps to reproduce

i will put a minimal reproduction project

### Minimal reproduction project

[example.zip](https://github.com/godotengine/godot/files/13461360/example.zip)

**Comment by nadako** [comment]
Yeah, GDScript types don't magically appear in .NET and I don't think there's an easy-to use mechanism to define "extern" classes to interface with GDScript in a typed way.

But you can access fields and methods like described [here](https://docs.godotengine.org/en/stable/tutorials/scripting/cross_language_scripting.html#accessing-gdscript-fields-from-c)

If you change your code like this, it works:

```csharp
	public override void _Ready()
	{
		var Globals = GetNode("/root/Globals");
		GD.Print("test is " + Globals.Get("test"));
	}
```
